Formula/bali-phy.rb

```diff
@@ -22,7 +22,7 @@ class BaliPhy < Formula
   def install
     flags = %w[-C build install]
     # Reduce memory usage for Circle CI
-    flags += %w[-j 4] if ENV["CIRCLECI"]
+    flags += %w[-j 2] if ENV["CIRCLECI"]
```

Comment:
You'll need to change `CIRCLECI` to `CI` for now. I've opened a PR https://github.com/Homebrew/brew/pull/5402 to address it.

Reply:
PR https://github.com/Homebrew/brew/pull/5402 was merged.